Remove MakeWatt, and added a blurb about idiomatic data constructors (#289)
purescript-contrib/purescript-book: text/chapter5.md

{{#include ../exercises/chapter5/src/ChapterExamples.purs:Coulomb}}
```

In this case, `Coulomb` is the _type constructor_ and `MakeCoulomb` is the _data constructor_. These constructors live in different namespaces, even when the names are identical, such as with the `Volt` example. This is true for all ADTs. Note that although the type constructor and data constructor can have different names, in practice it is idiomatic for them to share the same name. This is the case with `Amp` and `Volt` types above.

Another application of newtypes is to attach different _behavior_ to an existing type without changing its representation at runtime. We cover that use case in the next chapter when we discuss _type classes_.

## Exercises

> 1. (Easy) Use `newtype` to define a `Watt` unit for `Number` also using `Watt` as the data constructor, then define a `calculateWattage` function using this new `Watt` type and the above definitions `Amp` and `Volt`:

Reviewer: ```suggestion
1. (Easy) Define `Watt` as a `newtype` of `Number`. Then define a `calculateWattage` function using this new `Watt` type and the above definitions `Amp` and `Volt`:
```
